edna.less

```diff
@@ -3,11 +3,11 @@
 ****************/
 
 // REFERENCE FILES
-@import (reference) 'less/_mixins.less';
 @import (reference) 'less/_variables.less';
 
 // BASE FILES
 @import 'less/_normalize.less';
+@import 'less/_mixins.less';
```

Comment:
This change has existed in Buzz for a bit already, I'm just moving it here. The mixins file contains some utility classes that are used directly in the HTML.